services/scraper/src/server.ts

```
@@ -29,26 +35,507 @@ app.use((req, res, next) => {
+      case 'retry':
+        result = await errorHandler.forceRecovery(errorId, async () => {
+          return { recovered: true, timestamp: new Date() };
+        });
+        break;
+        
+      case 'skip':
+        result = { skipped: true, errorId, timestamp: new Date() };
+        break;
+        
+      case 'details':
+        const recentErrors = errorHandler.getRecentErrors(100);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)